src/js/smtpserver.h

```diff
@@ -3,6 +3,25 @@
 
 #include "js.h"
 
+#define	TRUE	1
+#define	FALSE	0
+
 int js_smtp_server_obj_init(JSContext *cx, JSObject *global);
 
+// Creates Javascript Object with response
+jsval create_response(JSContext *cx, int status, const char* message, int disconnect);
+
+// C stub handlers
+JSBool smtpInit(JSContext *cx, unsigned argc, jsval *vp);
```

Comment:
Stubs should not be exposed outside the JS module (and should be static, by the way) since they are used only internally by the JS module to initialize the `smtpServer` object.